tests/components/yardian/__init__.py

```diff
@@ -0,0 +1 @@
+"""Tests for the yardian component."""
```

Comment:
Either add tests or remove this file.

Reply:
I'll move this change to a new PR

Reply:
> Tests are already there, but the `__init__.py` was missing.
> 
> This is technically a bug fix so it should be its own PR

Sorry did miss this on github :)

Reply:
I only noticed it because I missed it in the initial pr 🙈